modules/infrastructure/permissions/cloud-connector/main.tf

```diff
@@ -26,31 +25,4 @@ data "aws_iam_policy_document" "cloud_connector" {
     ]
     resources = [var.cloudtrail_subscribed_sqs_arn]
   }
-
-
-  statement {
-    sid    = "AllowReadSecurityHub"
-    effect = "Allow"
-    actions = [
-      "securityhub:GetFindings",
-      "securityhub:BatchImportFindings",
-    ]
-    resources = ["arn:aws:securityhub:${data.aws_region.this.name}::product/sysdig/sysdig-cloud-connector"]
-    # TODO. make an input-var out of this
-  }
-
-
-  statement {
-    sid    = "AllowCloudwatchLogManagement"
-    effect = "Allow"
-    actions = [
-      "logs:CreateLogStream",
-      "logs:DescribeLogStreams",
-      "logs:GetLogEvents",
-      "logs:FilterLogEvents",
-      "logs:PutLogEvents",
```

Comment:
Remove only this one, the others are still requried for EKS

Reply:
removed all permissions that require write perm (create and put)